### ISSUE TITLE
Task Notification still popping in the web interface even when reset the User permission to default?

### ISSUE THREAD

**Issue by rainersg** [frappe/erpnext#4684]
First of all, I did not do anything to the Task/ToDo users and role permissions. But whenever I (the system admin) assign the task to one of the users, the rest of the users get the notifications. 

And I see that it can be done in the Role Permission. However, there is no guide/steps available to adhere to, to get the ideal scenario. I would like to ask not to close it abruptly after saying how it works without explaining in depth. And I even posted in discuss.erpnext.com but there is no reply or even single reply without a detailed explanation. 

Related post: https://github.com/frappe/erpnext/issues/4673. I know I should not post duplicated threads but what to do, the author suddenly closed it with a reply which I dont feel that it is helpful to me.

**Comment by rmehta** [comment]
@rainersg maybe you are using a group email id, I cannot replicate it.

**Comment by rainersg** [comment]
Hi @rmehta, can you explain what the group email id is? As far as I remember, I did not set anything in the ERPNext, other than adding the tasks/projects. Can you kindly point where the group email id is at? I will see if I am able to explain the issue.

**Comment by rmehta** [comment]
Must be something in your email setting, you may have set redirects. Make a trial account at erpnext.com and try and reproduce this.

**Comment by rainersg** [comment]
@rmehta i have signed up and did the email account setting. But nothing happened. Normally, when I set up the email account in the company erpnext using my email, it worked. However when I tried to set up using my email address in the trial erpnext, it did not work.

Anyway, what are the redirects about? As far as I remember, what I did was to set up the email account and then continued with the projects/tasks. That's it.
